### PR TITLE
[tracker] Adding tracker documentation.

### PR DESCRIPTION
**Pull request by ronnyvotel**
This is standalone tracker documentation, which individual models can reference in their own documentation.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/783)

### CODE REVIEW

File: pose-detection/src/calculators/tracker.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+A tracker is an algorithm that (typically) runs downstream of detection. The
+tracker maintains a history of objects (i.e. *tracks*), and its objective is to
+*link* incoming detections with these tracks (and spawn new tracks or kill old
+ones).
+
+To adequately match incoming detections with preexisting tracks, we must have a
+measure that scores the similarity:
+
+<img src="https://latex.codecogs.com/png.image?\dpi{300}&space;\mathrm{score}&space;=&space;\mathrm{similarity\_fn}(\mathrm{detection},&space;\mathrm{track})" title="\mathrm{score} = \mathrm{similarity\_fn}(\mathrm{detection}, \mathrm{track})" width="300"/>
```
Comment (ardoerlemans) [inline pose-detection/src/calculators/tracker.md:23]: I'm not familiar with including math in HTML like this, but can we be sure this link never breaks? And did you verify that the latex in the title actually works?

File: pose-detection/src/calculators/tracker.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  <img src="../../assets/tracker_example.png" alt="drawing" width="600"/>
+</p>
+
+Note that one detection is successfully linked with a track, while the other is
+not linked, due to a very low IoU. In this case, a new track is created for the
+unmatched detection. The track that was not linked stays in storage,
+but has the potential to be deleted in the future if no detections are
+associated with it.
+
+To build your own tracker, subclass `Tracker` in
```
Comment (ardoerlemans) [inline pose-detection/src/calculators/tracker.md:50]: If users want to build their own tracker, it does mean they have to checkout the pose detection code, update things like TrackerConfig and then build a new package.. Is that what you intended here?

**Review by ardoerlemans — COMMENTED** [review]
Looks good! I left two comments.

**Review by lina128 — APPROVED** [review]
Thank you for the documentation, @ronnyvotel, very helpful!

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/783#-:-MglxlRGEfr2n4lbfiXd:b-g05gvz)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @ardoerlemans and @ronnyvotel)

___
*[pose-detection/src/calculators/tracker.md, line 23 at r3](https://reviewable.io/reviews/tensorflow/tfjs-models/783#-MglA8u3F6N-W1iX3qt6-r3-23:-MglxDUN96zI-WNp9NrM:bmbfe5o) ([raw file](https://github.com/tensorflow/tfjs-models/blob/5ce12e2e163b58be6cebaf7801276ca7aa91494a/pose-detection/src/calculators/tracker.md#L23)):*
<details><summary><i>Previously, ardoerlemans (Ard Oerlemans) wrote…</i></summary><blockquote>

I'm not familiar with including math in HTML like this, but can we be sure this link never breaks? And did you verify that the latex in the title actually works?
</blockquote></details>

similarity_score)
___
*[pose-detection/src/calculators/tracker.md, line 41 at r3](https://reviewable.io/reviews/tensorflow/tfjs-models/783#-MglxVja8GAyXqB-V4Ef:-MglxVja8GAyXqB-V4Eg:b56k4fy) ([raw file](https://github.com/tensorflow/tfjs-models/blob/5ce12e2e163b58be6cebaf7801276ca7aa91494a/pose-detection/src/calculators/tracker.md#L41)):*
> ```
> <p align="center">
>   drawing
> </p>
> ```
tracker_example
___
*[pose-detection/src/calculators/tracker.md, line 50 at r3](https://reviewable.io/reviews/tensorflow/tfjs-models/783#-MglA8u3F6N-W1iX3qt6-r3-50:-MglwTDJ6uUKDwu2vquq:b-c8jwyj) ([raw file](https://github.com/tensorflow/tfjs-models/blob/5ce12e2e163b58be6cebaf7801276ca7aa91494a/pose-detection/src/calculators/tracker.md#L50)):*
<details><summary><i>Previously, ardoerlemans (Ard Oerlemans) wrote…</i></summary><blockquote>

If users want to build their own tracker, it does mean they have to checkout the pose detection code, update things like TrackerConfig and then build a new package.. Is that what you intended here?
</blockquote></details>

That's fine, we do have community contributors who contribute code to our codebase. It's an open source project. Also, Google researchers like you contribute from time to time. It's nice that they have some documentation to follow.

**Review by lina128 — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/783)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @ardoerlemans and @ronnyvotel)

___
*[pose-detection/src/calculators/tracker.md, line 23 at r3](https://reviewable.io/reviews/tensorflow/tfjs-models/783#-MglA8u3F6N-W1iX3qt6-r3-23:-Mgly53q0yURjHAmDIcA:bbj5hon) ([raw file](https://github.com/tensorflow/tfjs-models/blob/5ce12e2e163b58be6cebaf7801276ca7aa91494a/pose-detection/src/calculators/tracker.md#L23)):*
<details><summary><i>Previously, lina128 (Na Li) wrote…</i></summary><blockquote>

similarity_score)
</blockquote></details>

I just checked how it looks in the webpage, works perfectly. You can ignore my comment.
___
*[pose-detection/src/calculators/tracker.md, line 41 at r3](https://reviewable.io/reviews/tensorflow/tfjs-models/783#-MglxVja8GAyXqB-V4Ef:-MglyCOm222_xvlWJ2YZ:bbj5hon) ([raw file](https://github.com/tensorflow/tfjs-models/blob/5ce12e2e163b58be6cebaf7801276ca7aa91494a/pose-detection/src/calculators/tracker.md#L41)):*
<details><summary><i>Previously, lina128 (Na Li) wrote…</i></summary><blockquote>

> ```
> <p align="center">
>   drawing
> </p>
> ```
tracker_example
</blockquote></details>

I just checked how it looks in the webpage, works perfectly. You can ignore my comment.

**Pull request merged**